nextcloud/mail
# feat: Easy unsubscribe from lists with http unsubscribe header

Contributes part 1/2 of https://github.com/nextcloud/mail/issues/5387 for lists that send an unsubscribe URL, **not** an unsubscribe mailto URL.

| 1 | 2 | 3
|---|---|---|
|![Bildschirmfoto vom 2023-04-27 08-53-11](https://user-images.githubusercontent.com/1374172/234782926-25c8f144-b466-40bd-bbde-d141ebc0efeb.png) |![Bildschirmfoto vom 2023-04-27 08-52-21](https://user-images.githubusercontent.com/1374172/234782843-3bc4d2bc-c13c-4723-987f-3e791482c3d2.png) | ![Bildschirmfoto vom 2023-04-25 09-59-14](https://user-images.githubusercontent.com/1374172/234215443-f1c9f598-bc21-45d8-aacb-331e20e34eea.png)

## Todo

- [x] Merge https://github.com/nextcloud/mail/pull/8376
- [x] Merge https://github.com/nextcloud/mail/pull/8416
- [x] Move action button

## How to test

1) Set up an account in Mail
2) Open a message from a mailing list (it has to have a `List-Unsubscribe: <http…>` header
3) See the *Unsubscribe* button
4) Click the *Unsubscribe* button and see the mailing list landing page in another tab
